docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3133,7 +3136,83 @@ needed in the noise functions for this component.
 
 .. todo::
 
-  Row based noise and data formatting noise to be added 
+  Duplicates, improper inclusion, and data formatting noise to be added 
+
+Row Noise
+'''''''''
+
+Row based noise is defined as the inclusion or exclusion of entire rows of 
+data – which in practice is including or excluding simulants. This falls 
+into three buckets:  
+
+#. **Omissions:** not including simulants that should be included  
+
+#. **Duplicates:** including some simulants multiple times in the data. This could be identical rows of data, or the same simulant but with some differences in their data.  
+
+#. **Improper inclusion:** including simulants in the dataset that don’t qualify for inclusion. This only applies to observers which are not meant to include everyone. An example is a simulant being included in WIC that is above the maximum income level.  
+
+**Omissions:**  
+
+It is assumed that in any dataset, there are people missed. This 
+is important in PRL as the user will try to find a match for someone 
+that doesn’t exist. For simplicity, we will have pseudo-people end 
+users input one value: the overall omission rate. This value can 
+range from 0% (indicating NO omission or a perfect response) up 
+to 10% omission and will be defined at the observer level.  
+
+For the census and household surveys, where non-response is built 
+into the observer, the default omission rate will the equal to the 
+expected omission rate of that survey. For the census and ACS survey, 
+this is 1.45%. For the CPS survey, this is 29.05%. The other observers’ 
```

Comment:
above you say "up to 10% omission"; is that incompatible with the CPS default of 29.05%?

Reply:
Great point. I changed this to 50%